tests/fixtures/seeds.json

```diff
@@ -862,6 +862,46 @@
     "repositoryId": "927e26f6-da29-40d6-b9de-530bdcddf1ed",
     "description": "a description"
   },
+  {
```

Comment:
I will update seed to avoid manual addition.

Reply:
Feel free to ping me when seed is updated!

Reply:
It's merged 👌

Reply:
Well, actually I still had tu update it as the color wasn't matching the svg otherwise 😅 Tell me if it's an issue

Reply:
I change seeds to use `AlmostBlack` explicitly in the stroke property of the vector `activity`